engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -1046,8 +1046,9 @@ public abstract class ProcessEngineConfigurationImpl extends ProcessEngineConfig
    * when it has never been enabled/disabled before.
    * Subsequent changes can be done only via the
    * {@link ManagementService#toggleTelemetry(boolean) Telemetry} API in {@link ManagementService}
+   * Telemetry is deactivated by default.
    */
-  protected Boolean initializeTelemetry = null;
+  protected Boolean initializeTelemetry = false;
```

Comment:
❓Could we change it to primitive boolean?

Reply:
yes, I think we can do it. I will push adjustment.
as it's impl, is not guaranteed backwards compatibility

Reply:
It can also be a follow-up if you are worried that it might break something.

Reply:
Maybe one test is broken if I can't fix it I will revert the switch.

Reply:
No test failures, feedback implemented.